### PR TITLE
Mobile menu date 12/21/2022

### PR DESCRIPTION
**Pull request by fithamlak**
=>I  implement all the following features:
  -When the user clicks (or taps) the hamburger button, the mobile menu appears.
 -When the user clicks (or taps) the close (X) button, the mobile menu disappears.
 -When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
 -When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

I was also on coding partner challenge 

image

### CODE REVIEW

**Comment by mirouhml** [comment]
Hi @fithamlak, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have a complete README, please know that all main headers from the [README template](https://github.com/microverseinc/readme-template) should be present, the only optional section is FAQ, also, make sure to add the section Future Features, including 1-3 potential features to be added.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: index.html [hunk 1]
```diff
@@ -24,17 +24,18 @@
...
+              <li class="menu-bar">
                 <a href="#works-container">Portfolio</a>
               </li>
-              <li class="mebu-bar">
+              <li class="menu-bar">
                 <a href="#about-container">About</a>
               </li>
-              <li class="mebu-bar">
+              <li class="menu-bar">
                 <a href="#footer-header">Contact</a>
```
Comment (mckent05) [inline index.html:39]: - Good job done so far 👏, but there are some differences between your project and the Figma template. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏.

Your project:
-- **OPTIONAL** - Please ensure your logo is centered in the middle of the navbar to help improve the UI of your website 👍 👍 
bjejaet

-- You have not styled the mobile menu the way it is in the design template, please remember to stick to the styles in the design (fonts, color, bg images, margins, et.c..). Please style your mobile menu so it looks close to the one in the Figma template 🙏 
egwrgwrg

Figma Design:
bqehfbqhe

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @fithamlak, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 
- Good job using the correct workflow for this milestone 👏 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by uwadonat — APPROVED** [review]
## Hi @fithamlak,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉 
### Highlights
- No linter errors ✔️
- Clean and well organized project ✔️
awoseme


# Status: Approved ✔️✔️✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by fithamlak** [comment]
mobile menu pop up is done, thank you for your constructive feedback.

**Review by Abenezer-Tilahun — COMMENTED** [review]
Hi @fithamlak 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Github-Flow.
✔️ Professional README  
✔️ Good Commit Messages
✔️ Good job for Implemented Portfolio - mobile menu

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**